dusty/cli/__init__.py

```diff
@@ -82,7 +82,9 @@ def _run_command(sock, command):
         if data:
             stripped = data.decode('utf-8').replace(constants.SOCKET_TERMINATOR, '').replace(constants.SOCKET_ERROR_TERMINATOR, '')
             sys.stdout.write(stripped)
-            if data.endswith(constants.SOCKET_TERMINATOR):
+            if stripped == constants.SOCKET_ACK:
```

Comment:
I think the complete answer there is a little complicated but with this tiny data size I am seeing that that's how it's working in practice